docs/faq.md

```diff
@@ -179,3 +179,9 @@ In the future the OpenShift installer will likely have support for this.
 ## Q: Can I use Driver Update Program disks with RHCOS?
 
 No, there is no supported mechanism for non-default kernel modules at this time, which includes [driver disks](https://access.redhat.com/articles/64322).
+
+## Q: How do I capture console logs from an RHCOS node running on vSphere?
+
+It's possible to write the serial console data directly to the VMFS volume.  You can do this by changing the Virtual Hardware settings of the VM to include a serial port that writes to a file (see [screenshot](https://raw.githubusercontent.com/openshift/os/docs/vsphere-settings.png)).
```

Comment:
I'm not sure this link will work as expected. I _think_ a branch is required. As an example:

- No branch (404): https://raw.githubusercontent.com/openshift/os/docs/faq.md
- Branch: https://raw.githubusercontent.com/openshift/os/master/docs/faq.md

Reply:
Ah, yeah...quick and dirty like I said  :)

Will fix